### ISSUE TITLE
Mentions in comments are not resolved to user and no hover support

### ISSUE THREAD

**Issue by sheldonhull** [ankitbko/vscode-pull-request-azdo#48]
- Extension version: v0.0.22
- VSCode Version: 1.57.0
- OS: Darwin x64 20.4.0

After beginning report I noticed this at the bottom under known issues.
Could not find an open issue for this. 

Recommend highlighting this at the top of the readme rather than bottom to make it very clear upfront.

User name isn't being resolved with `@name` and is coming back as guid in rendering.

![2021-06-11-13 09 10-Code](https://user-images.githubusercontent.com/3526320/121731245-4b3bc900-cab6-11eb-82aa-9d72dc8b2993.png)

**Comment by ankitbko** [comment]
Yes this is known issue. I will need to think about how to implement this. The comment API does not provide any way to display a particular segment of string as some other string. That means even if I am able to correlate the id with the user, I will need to *modify* the actual comment content to replace the id with the user's name and display that. The problem arises what if the person *edits* the comment and submits. Then I will need to replace the name with the id before submitting the edit. This is same problem as mentioning a user in comment so this needs to be solved first. The problem there is to actually convert a "name" to the id.
Need to think about this a bit.
